sdk/storage/azblob/sas/service.go

```diff
@@ -103,7 +104,8 @@ func (v BlobSignatureValues) SignWithSharedKey(sharedKeyCredential *SharedKeyCre
 		string(v.Protocol),
 		v.Version,
 		resource,
-		snapshotTime,         // signed timestamp
+		snapshotTime, // signed timestamp
+		v.EncryptionScope,
 		v.CacheControl,       // rscc
 		v.ContentDisposition, // rscd
 		v.ContentEncoding,    // rsce
```

Comment:
As per the [documentation](https://learn.microsoft.com/en-us/rest/api/storageservices/create-service-sas#version-2020-12-06-and-later), `v.ContentType // rsct` is removed from version 2020-12-06 and later for creating signature string. You can check if this is actually the case or is it a documentation bug.

Reply:
I believe that is a bug, I tried to remove it and get 'Authentication Failed'. I double checked the string to sign in the .NET SDK and it matches with what we have here.

https://github.com/Azure/azure-sdk-for-net/blob/e8f203346cbc8135aac9ce250853fd4cbed9d75d/sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs#L350